controller/items.go

```diff
@@ -6,13 +6,41 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/gorilla/mux"
 	"github.com/rodrigomkd/go-rest-api/service"
+
+	"github.com/gorilla/mux"
 )
 
-func GetItems(w http.ResponseWriter, r *http.Request) {
+type IController interface {
+	GetItems(w http.ResponseWriter, req *http.Request)
+	GetItem(w http.ResponseWriter, req *http.Request)
+	GetItemsSync(w http.ResponseWriter, req *http.Request)
+}
+
+type Controller struct {
```

Comment:
You could be a bit more explicit about what this controller is, when importing this you would have `controller.Controller` and this doesn't really tell much about what information it handles, I would suggest changing it to `ItemsController`